composefiles/swarm-uniconfig.yml

```diff
@@ -18,6 +18,11 @@ x-logging: &logging_loki
         - action: labelmap
           regex: swarm_(service)
 
+x-placement: &placement
+      placement:
+        constraints:
+          - node.role == manager
```

Comment:
should uniconfig be on manager node ?

Reply:
Default settings are for single-node deployment and all services are running on a master node.
This parameter is replaced during generation compose files by selected placement type.